enhancements/console/quick-starts.md

```diff
@@ -151,6 +154,23 @@ resource actions. In order for the user to see the Quick Start and
 take its tour, his access review needs to pass all the listed
 resources actions.
 
+#### Quick Start Internationalization
+
+OpenShift can be toggled between multiple languages. The
+Quick Start CRD `spec` contains a `language` field to specify what
```

Comment:
So this goes with `Have separate ConsoleQuickStart resources for each language and use a label to indicate the language.` approach. Right ?